ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml

```diff
@@ -291,13 +275,11 @@ tests:
     cluster_profile: aws-perfscale
     env:
       BASE_DOMAIN: perfscale.devcluster.openshift.com
-      CILIUM_VERSION: 1.14.4
+      CILIUM_VERSION: 1.15.1
       COMPUTE_NODE_REPLICAS: "120"
       COMPUTE_NODE_TYPE: m5.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m5.4xlarge
       HUBBLE: "true"
-      KUBE_BURNER_VERSION: 1.8.0
-      PROFILE_TYPE: reporting
```

Comment:
do we want to remove the profile type here?

Reply:
yes - given this job only runs once a month.